starter-code/bootstrap-cloning-revera.html

```diff
@@ -0,0 +1,107 @@
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        
+        <div class="collapse navbar-collapse" id="navbarSupportedContent" class="dropdown-menu" id="dropdown-list">
+            <ul class="navbar-nav mr-auto">
+                
+                <li class="nav-item active">
+                    <a class="nav-link" href="#">Home</a>
+                </li>
+
+                <li class="nav-item">
+                    <a class="nav-link" href="#">About</a>
+                </li>
+
+                <li class="nav-item">
+                    <a class="nav-link" href="#">Sample page</a>
+                </li>
+
+                <li class="nav-item">
+                    <a class="nav-link" href="#">Portfolio</a>
+                </li>
+
+                <li class="nav-item">
+                    <a class="nav-link" href="#">Blog</a>
+                </li>
+
+                <li class="nav-item dropdown">
+                    <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                        Parent page >>
+                    </a>
+                    <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+                        <a class="dropdown-item" href="#">Full page</a>
+                        <div class="dropdown-divider"></div>
+                        <a class="dropdown-item" href="#">Child page 1</a>
+                        <div class="dropdown-divider"></div>
+                        <a class="dropdown-item" href="#">Child page 2</a>
+                    </div>    
+                </li>
+
+        </div>
+    </nav>
+
+    <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
+        <ol class="carousel-indicators">
+            <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
+            <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
+            <li data-target="#carouselExampleIndicators" data-slide-to="2"></li>
+        </ol>
+        
+        <div class="carousel-inner">
+            
+            <div class="carousel-item active">
+                <img class="d-block w-100" src="./img/slider-1.jpg" alt="First slide">
+            </div>
+            
+            <div class="carousel-item">
+                <img class="d-block w-100" src="./img/slider-2.jpg" alt="Second slide">
+            </div>
+            
+            <div class="carousel-item">
+                <img class="d-block w-100" src="./img/slider-3.jpg" alt="Third slide">
+            </div>
+            
+            <div class="carousel-item">
+                <img class="d-block w-100" src="./img/slider-4.jpg" alt="Fourth slide">
+            </div>
+
+            </div>
+                <a class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev">
```

Comment:
Arrows are not working for me. Bootstrap is also using jQuery, so you may have something broken related to this jquery.